modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -152,6 +152,17 @@ Several  xref:clauses/transaction-clauses.adoc#query-listing-transactions[`SHOW
 * `startTime` and `currentQueryStartTime` now return a `ZONED DATETIME` instead of a `STRING` representation of a temporal value.
 * `clientAddress` and `outerTransactionId` now return `null` instead of an empty `STRING` when unavailable.
 * The current query-related columns — `currentQuery`, `currentQueryId`, `parameters`, `planner`, `runtime`, `indexes`, `currentQueryStartTime`, `currentQueryElapsedTime`, `currentQueryCpuTime`, `currentQueryIdleTime`, and `currentQueryStatus` — now return `null` when no query is executing.
+
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+REVOKE READ {*} ON GRAPH * NODES A FROM regularUser
+REVOKE ROLE regularUser FROM bob
+----
+a|
+Notifications for impossible `REVOKE` commands, where a user, a role, or a database does not exist, have been replaced with errors.
```

Comment:
I feel like maybe it should be under `removed features` instead of `updated features`, we did remove the ability to not get an error...

Reply:
hmm, maybe. Feels more like a change than an removal (since we are changing the notification into an error). But I don't have a strong opinion 🤷

Reply:
Had we not had a notification before this would absolutely be a case of removed capabilities -> introducing an error, so I think I'd go with that here as well 🤔 errors are not really part of the features capabilities but rather things it can't do (and the updates section is usually for additional capabilities, not restricting it I think)

But we could see if the docs team has any other view on things